src/index.js

```diff
@@ -9,5 +9,61 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Player from '../src/Player'
+import Game from '../src/Game'
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let nameInputs = document.querySelector(".player-name-input");
+let gameRules = document.querySelector(".game-rules");
+let player1Input = document.querySelector(".player1");
+let player2Input = document.querySelector(".player2");
+let player3Input = document.querySelector(".player3");
+let player1Name = document.querySelector(".player1-name");
+let player2Name = document.querySelector(".player2-name");
+let player3Name = document.querySelector(".player3-name");
+let continueBtn = document.querySelector(".continue-button");
+let gameBoard = document.querySelector(".game-board");
+let playBtn = document.querySelector(".play-button");
+let player1;
+let player2;
+let player3;
+
+nameInputs.addEventListener("keyup", checkInputs);
+continueBtn.addEventListener("click", instantiatePlayers);
```

Comment:
This is also a good place for event delegation using jquery

Reply:
I agree... I think our first goal of implementing jQuery should be to handle DOM manipulation functionality. If we feel comfortable doing that, we could look into some data manipulation, but changing from JS to jQuery for DOM stuff shouldn't be too impossible